enodi/post_it
# #157429577 Design dashboard page

#### What does this PR do?
This PR designs the dashboard page

#### Description of Task to be completed?
- Ensures authenticated users see a dashboard page on successful sign-in.

#### How should this be manually tested?
- Pull this branch and run `foreman start -f Procfile.dev -p 3000`
- Navigate to `localhost:3000`.
- Sign into the application, you'll see a dashboard page.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[Story ID: 157429577](https://www.pivotaltracker.com/n/projects/2166405/stories/157429577)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29783644/39846033-60888760-53f1-11e8-93e3-c8c2963cfbb9.png)
